webapp/src/pages/ChatroomPage/ChatroomPage.tsx

```diff
@@ -32,9 +32,14 @@ export default function ChatroomPage() {
 type ChatroomProps = {
   chatroomId: string;
   authUser: UserWithAuthorizationResponseDto;
+  overridePermissions?: boolean;
 };
 
-function Chatroom({ chatroomId, authUser }: ChatroomProps) {
+function Chatroom({
+  chatroomId,
+  authUser,
+  overridePermissions = false,
```

Comment:
This is a way to have the two setups: normal, where it works as it should for regular users, even if the user is admin (no magic powers), and elevated privileges so that it can be seen fully when navigating to this page from the admin tab

Reply:
praise: I like this :+1: